announcements/1.16.1.md

```diff
@@ -0,0 +1,24 @@
+### Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.16.1.
+
+Please see the [component versions](https://anbox-cloud.io/docs/component-versions) for a list of updated components.
+
+#### New features & improvements
+
+* Bootstrapping Anbox Cloud on AWS requires the use of Instance Metadata Service[https://docs.aws.amazon.com/zh_cn/AWSEC2/latest/UserGuide/configuring-instance-metadata-service.html]. Since 1.16.1 release, Anbox Cloud makes use of `IMDSv2` rather than `IMDSv1` for the protection against Server Side Request Forgery (`SSRF`) attack. See the (link)[https://aws.amazon.com/blogs/security/defense-in-depth-open-firewalls-reverse-proxies-ssrf-vulnerabilities-ec2-instance-metadata-service/] for more information.
```

Comment:
```suggestion
* Bootstrapping Anbox Cloud on AWS requires using the [Instance Metadata Service](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/configuring-instance-metadata-service.html) (`IMDS`). Starting with the 1.16.1 release, Anbox Cloud uses `IMDSv2` rather than `IMDSv1` for protection against Server Side Request Forgery (`SSRF`) attacks. See the [AWS Security Blog](https://aws.amazon.com/blogs/security/defense-in-depth-open-firewalls-reverse-proxies-ssrf-vulnerabilities-ec2-instance-metadata-service/) for more information.
```